src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -817,6 +817,11 @@ uct_ib_mlx5_iface_select_sl(uct_ib_iface_t *iface,
 
     ucs_assert(iface->config.sl == UCT_IB_SL_INVALID);
 
+    if (uct_ib_device_is_port_roce(uct_ib_iface_device(iface),
```

Comment:
should we set sl to 0 ?

Reply:
maybe better to keep it set to `UCT_IB_SL_INVALID`?

Reply:
where would it be set to 0 for RoCE?

Reply:
> where would it be set to 0 for RoCE?

why it has to be set to `0`? it should be used for RoCE